src/web-auth/captcha.js

```diff
@@ -0,0 +1,117 @@
+// eslint-disable-next-line no-unused-vars
+import Authentication from '../authentication';
+import object from '../helper/object';
+
+var defaults = {
+  lang: 'en',
+  templates: {
+    'auth0': function (challenge) {
+      var message = challenge.type === 'code' ?
+        'Enter the code shown above' :
```

Comment:
@stevehobbsdev shouldn't these be obtained through some i18n method?

Reply:
there is no i18n in this repo afaik